CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKIconsRenderable.java

```diff
@@ -421,6 +415,29 @@ public void CopyTo(ByteBuffer buffer) {
         }
     }  
     
+    private static class Position {
+        private static final int SIZEOF = 2 * 4 * Float.BYTES;
+        public final Vector4f position1;
```

Comment:
how come a position is a vector4f?

Reply:
Is this actually a point? Or is radius like a hit radius or something

Reply:
Read down and you will see element 4 is radius.